src/templates/demo.js

```diff
@@ -1,20 +1,23 @@
 import React from "react";
-import { graphql } from "gatsby";
+import { graphql, Link } from "gatsby";
 import LayoutCodepen from "../components/layoutFullWidthWithCodepen";
 import PrevNextLinks from "../components/prevNextPostLinks/prevNextLinks";
+import Seo from "../components/seo";
 
 export default ({ data, pageContext }) => {
   return (
     <LayoutCodepen>
-				<h1>{data.markdownRemark.frontmatter.title}</h1>
-        <article dangerouslySetInnerHTML={{ __html: data.markdownRemark.html }} />
-				<PrevNextLinks prev={pageContext.prev} next={pageContext.next}></PrevNextLinks>
+			<Seo title={data.demoPage.frontmatter.title}></Seo>
+			<nav id="breadcrumb-nav" aria-label="breadcrumb"><Link to='/demos'>Demos</Link> > <span>{data.demoPage.frontmatter.title}</span></nav>
```

Comment:
shouldn't `>` proberly added as `&gt;` or is something that react does on its own?

Reply:
Babel seems to handle it for us!
**The JSX**
```HTML
<span> &gt; > </span>
```
**After Babel transpiles it**
```javascript
React.createElement("span", null, " > > ");
```
Which means, using either `&gt;` or **>**, would work! 😊